components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -317,13 +318,26 @@ class BrowserToolbar @JvmOverloads constructor(
         contentDescription: String,
         visible: () -> Boolean = { true },
         @DrawableRes background: Int? = null,
+        private val padding: Padding = Padding(
```

Comment:
Yes but we additionally need this in:
- `ToggleButton`
- `ActionImage`

Also, is there any reason this shouldn't be support in the lowest level class, like `Toolbar.ActionButton`, rather than `BrowserToolbar.Button`?

Reply:
I updated the PR with the changes!